shell/shared/grain.js

```diff
@@ -126,13 +124,6 @@ if (Meteor.isServer) {
       promise.cancel();
     });
   });
-
-  function cleanupExpiredTokens() {
```

Comment:
For offer templates, the whole point of having these expire was to clear them out of the database, since a new token is created every time the user looks at the page with the template. So I think we still want to clear expired tokens...